roles/open_zaak_k8s/templates/nginx.yml.j2

```
@@ -70,3 +70,6 @@ spec:
             mountPath: /app/private-media
             subPath: openzaak/{{ openzaak_instance }}/private-media
 {% endif %}
+      serviceAccountName: "{{ openzaak_service_account| default('default') }}"
+      securityContext:
+        fsGroup:  {{ openzaak_pod_security_context.fsGroup }}
```

```suggestion
      securityContext: {{ openzaak_pod_security_context }}
```
so then all aspects can be managed of the security context if needed.

---

Done, added RunAsUser and RunAsgroup (1000) to defaults. Now the nginx pod actually runs with these ID's (instead of 0).